src/wasm-lib/kcl/src/lsp/kcl/mod.rs

```diff
@@ -445,7 +445,10 @@ impl Backend {
                 let token_index = Arc::new(Mutex::new(token_type_index));
                 let modifier_index: Arc<Mutex<u32>> = Arc::new(Mutex::new(0));
                 crate::walk::walk(&ast, &|node: crate::walk::Node| {
-                    let node_range: SourceRange = (&node).into();
+                    let Ok(node_range): Result<SourceRange, _> = (&node).try_into() else {
```

Comment:
this is an unfortunate casualty of `KclNone` being in the ast, the `Into` is now a `TryInto`, forcing this `let` destructure.